nova-server/DataTypes.d.ts

```diff
@@ -0,0 +1,35 @@
+import { firestore } from 'firebase-admin';
```

Comment:
It seems to include changes from #14. Similar to #14, I'm not sure why we need this.

Reply:
My mistake, most likely branched off the wrong branch. I've removed the `nova-server` directory from this branch